src/lib.rs

```diff
@@ -68,15 +68,13 @@ pub fn leftpad_with<'a, S>(string: S, len: usize, pad_char: char) -> Cow<'a, str
 /// # Examples
 ///
 /// ```
-/// use left_pad::leftpad;
+/// use left_pad::{leftpad,leftpad_with};
```

Comment:
Did you actually get an error here? Because the commit says 'Fix doc-test error', and I they worked fine for me. Just curious though, I don't really mind moving the imports together.